apps/admin/backend/schema.sql

```diff
@@ -101,7 +101,11 @@ create table ballot_images (
 create table system_settings (
   -- enforce singleton table
   id integer primary key check (id = 1),
-  are_poll_worker_card_pins_enabled boolean not null default false
+  are_poll_worker_card_pins_enabled boolean not null,
+  inactive_session_time_limit_minutes integer not null,
+  num_incorrect_pin_attempts_allowed_before_card_lockout integer not null,
+  overall_session_time_limit_hours integer not null,
+  starting_card_lockout_duration_seconds integer not null
```

Comment:
I went back and forth on whether to make these nullable but ultimately decided not to since, from a user perspective, omitting any of the above settings in the system settings file could wrongly suggest that you're disabling the relevant feature entirely.

For example, omitting `inactiveSessionTimeLimitMinutes` might make you think that you've disabled the inactive session time limit altogether, but 1) we don't want to support that and 2) the auth lib uses the default whenever a setting is undefined (this is useful as a safeguard but also necessary for the case that a machine hasn't been configured with a system settings file yet).